docs/source/managed-federation/federated-schema-checks.mdx

```diff
@@ -2,44 +2,33 @@
+In particular, along with validating overall schema changes against known operations, running `rover subgraph check` for a subgraph helps ensure that all subgraph schemas still compose to a valid supergraph schema, and will output any violated dependencies if present.
 
-With a federated graph, use the `apollo service:check` command to validate individual service changes by adding the `--serviceName` flag.
-
-When you run `apollo service:check` on a federated service, Apollo Studio runs composition on the proposed capabilities with the current list of federated services and their capabilities, making sure that the composition is successful. That composed schema is then compared against the most recently registered schema to confirm that those changes are safe. If composition fails, then validation ends and results are returned to the user. Note that running `apollo service:check` never updates the graph.
+When you run `rover subgraph check` on a particular subgraph, Apollo Studio runs composition on the proposed capabilities with the current list of subgraphs and their capabilities, making sure that the composition is successful. That composed schema is then compared against the most recently registered schema to confirm that those changes are safe. If composition fails, then validation ends and results are returned to the user. Note that running `rover subgraph check` never updates the graph.
 
 There are two types of failures that can occur during validation: failed usage checks and failed composition. Failed usage checks are failures due to breaking changes, like removing a field that an active client is querying. Failed composition, on the other hand, is a failure due to inability to compose the graph, like missing a `@key` for an extended entity.
 
 ## Handling composition failure
 
-In general, an `apollo service:push` should only be run after an `apollo service:check` has passed, but even so, due to changes in _other_ services, it's possible that the `apollo service:push` command will encounter composition errors. When this happens, the federated service will still be updated as long as its capabilities are spec-compliant, but **the graph will not be updated**. This means that a new schema will not be associated, nor will the gateway's configuration be updated.
+In general, you should run `rover subgraph publish` only after a successful run of `rover subgraph check`. Even in this case, however, it's possible that `rover subgraph publish` will encounter composition errors because of simultaneous changes to _another_ subgraph. When this happens, your subgraph's registered schema will still be updated as long as its capabilities are spec-compliant, but **the graph will not be updated**. This means that a new schema will not be associated, nor will the gateway's configuration be updated.
```

Comment:
Suggestion: change "When this happens" to "When a composition error occurs after publishing a subgraph" to be less ambiguous.